src/App.tsx

```diff
@@ -15,71 +17,94 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  HandlerSelectedGoods = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  HandlerRemoveGood = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  render() {
+    const { selectedGood } = this.state;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        <h1 className="title is-flex is-align-items-center">
+          {!selectedGood
```

Comment:
```suggestion
          {selectedGood
```
It's better to check if good is present in order not to confuse other developers

Reply:
Thank you, Oleksandr.